specification/sql/resource-manager/readme.md

```diff
@@ -1028,7 +1027,7 @@ input-file:
 - ./Microsoft.Sql/preview/2020-11-01-preview/FailoverGroups.json
 - ./Microsoft.Sql/preview/2020-11-01-preview/FirewallRules.json
 - ./Microsoft.Sql/preview/2020-11-01-preview/InstancePools.json
-- ./Microsoft.Sql/preview/2020-11-01-preview/JobAgents.json
+- ./Microsoft.Sql/preview/2023-02-01-preview/JobAgents.json
```

Comment:
I'm puzzled about
why is this file changing like this, if the PR is for adding eatures to 2023-05-01-preview?